packages/client/src/sync/fetcher/storagefetcher.ts

```diff
@@ -129,13 +127,17 @@ export class StorageFetcher extends Fetcher<JobTask, StorageData[][], StorageDat
       )
       const keys = slots.map((slot: any) => slot.hash)
       const values = slots.map((slot: any) => slot.body)
-      return await this._proofTrie.verifyRangeProof(
+      return await Trie.verifyRangeProof(
         stateRoot,
         origin,
         keys[keys.length - 1],
         keys,
         values,
-        <any>proof
+        <any>proof,
```

Comment:
Is there a strong reason we need to cast this to `any` instead of just type casting it as we expect (either as `Uint8Array[]` if we always expect it to be bytes or else `Uint8Array[] | Undefined`)?

Reply:
I've changed it so that the value is converted to the expected type instead of casting as any.